web/app/.eslintrc

```diff
@@ -0,0 +1,51 @@
+{
+  "plugins": [
+    "eslint-plugin-react", // https://github.com/yannickcr/eslint-plugin-react
+    "eslint-plugin-promise"
+  ],
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "env": {
+    "browser": true,
+    "es6": true
+  },
+  "extends" : ["eslint:recommended", "plugin:react/recommended"],
+  "rules": {
```

Comment:
I'd also like to add `"indent": ["error", 2]` to the rules at some point. It does change a lot of files, so I understand if you want to hold off on including it in this branch. I'm sure there's a bunch of other rules we can add after this is set up too!